core/src/main/java/de/javakaffee/web/msm/MemcachedNodesManager.java

```diff
@@ -184,6 +187,12 @@ public static MemcachedNodesManager createFor(final String memcachedNodes, final
 		if ( memcachedNodes == null || memcachedNodes.trim().isEmpty() ) {
 			throw new IllegalArgumentException("null or empty memcachedNodes not allowed.");
 		}
+		
+		if (memcachedNodes.startsWith("redis://") || memcachedNodes.startsWith("rediss://")) {
```

Comment:
What's this "rediss" for? Perhaps add a comment here is it has special semantics?